tools/restatectl/src/commands/metadata_server/status.rs

```diff
@@ -8,10 +8,10 @@
 // the Business Source License, use of this software will be governed
 // by the Apache License, Version 2.0.
 
-use std::collections::BTreeMap;
-
+use bytesize::ByteSize;
 use futures::future::join_all;
 use itertools::Itertools;
+use std::collections::BTreeMap;
```

Comment:
would be nice if we kept original import organization

Reply:
Sorry, it's me not being able to tell RustRover to not do these changes 🙈